server/data/quotes.json

```
@@ -303,6 +303,10 @@
     "author": "Kurt Tucholsky",
     "sentence": "Who wants to have an effect on others, must first speak in their language with them."
   },
+  {
+    "author": "Alexis Fledge",
+    "sentence": "A man who experienced more than one language has experienced more than one Culture."
```

Remove capitalization for Culture
```suggestion
    "sentence": "A man who experienced more than one language has experienced more than one culture."
```

---

did remove and inserted the quote into the dictionary order